packages/vm/lib/index.ts

```diff
@@ -89,6 +90,11 @@ export interface VMOpts {
    * Default: `false` [ONLY set to `true` during debugging]
    */
   allowUnlimitedContractSize?: boolean
+
+  /**
+   * Provide a non-canonical genesisBlock (non-canonical with respect to the `Common`) to initialize the blockchain with
+   */
+  genesisBlock?: Block
```

Comment:
I think I would prefer not to bloat the options here and do without this `genesisBlock` option, which to some extend replicates the `Blockchain` options. If someone wants to use this it is still possible to create ones own `Blockchain` object and pass this in.

Reply:
If I remove this how should I setup a blockchain with a custom genesis block? I am pretty keen on having the genesis being set in constructor.

Reply:
```typescript
const blockchain = Blockchain.create({ genesis })
const vm = new VM ({ blockchain })
```

(rough example). Or am I missing something?

Reply:
I am confused here, this is what is supported currently? It is only called `genesisBlock` instead of `genesis`?

Reply:
This discussion is about the genesisBlock option in the VM, are you aware of that?

Reply:
Ah woops, I was not 😅  Yes I get your point, I'll remove this from the VM options, consumers can indeed just pass on their custom `blockchain` 👍